modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -2418,7 +2481,7 @@ else if (s.contains("clojure")) {
     }
 
     /**
-     * Stops grid instance.
+     * Stops ignite instance.
```

Comment:
It seem that _Ignite_ with first capital letter is used elsewhere in this class docs.

Reply:
fixed.